<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addOvertimeForm</class>
 <widget class="QWidget" name="addOvertimeForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>368</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>318</width>
    <height>368</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>318</width>
    <height>368</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add overtime</string>
  </property>
  <widget class="QTextEdit" name="noteText">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>191</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>171</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Write some notes (if you need):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter ovetime hours:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="minimumDate">
    <date>
     <year>2017</year>
     <month>12</month>
     <day>23</day>
    </date>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::UTC</enum>
   </property>
   <property name="date">
    <date>
     <year>2018</year>
     <month>1</month>
     <day>20</day>
    </date>
   </property>
  </widget>
  <widget class="QSpinBox" name="hours_counter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
  </widget>
  <widget class="QDial" name="regulator">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string extracomment="Select overtimes hours"/>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose date:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancle</string>
   </property>
  </widget>
  <widget class="QLabel" name="overtime_level">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>88</width>
     <height>95</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/overtime_level1.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>regulator</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hours_counter</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hours_counter</sender>
   <signal>valueChanged(int)</signal>
   <receiver>regulator</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
